### ISSUE TITLE
Reviews menu missing when accessing Reviews from Hub Menu

### ISSUE THREAD

**Issue by jostnes** [woocommerce/woocommerce-ios#5872]
**Describe the bug**
Found this while updating the UI tests based on `trunk` branch, when the merchant first taps to view Reviews the top-right menu is missing. But will reappear after tapping on a review and returning back to the reviews list screen. 

**To Reproduce**
1. With Hub Menu turned on > Go to Menu > Reviews
2. See that the triple dot menu is missing on top right
3. Tap to view a review from the reviews list
4. Tap back to return to reviews list
5. See that the triple dot menu is now available

**Screencap**

https://user-images.githubusercontent.com/17252150/149081900-72068c8a-f39c-4371-a027-ef869903b46f.mp4

**Expected behavior**
The menu option should always be available

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: Simulator iPhone 11
- iOS version: iOS 15.0
- WooCommerce iOS version: 8.3

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by itsmeichigo** [comment]
Hi, @jostnes 👋 I think this issue may be related to #5830 - that issue was logged with technical details so it might not be obvious 😅 We'll make sure to fix this before the Hub menu feature is released (it's currently under a feature flag and this issue only happens when the Reviews screen is accessed from the Hub menu).

**Comment by jostnes** [comment]
Ah right, that looks to be the same issue. To be honest, I was mostly looking for screenshots and videos on issues with type "bug" and couldn't find one related to what I'm seeing 😅 

I'll mark this as a duplicate then and have the issue tracked in the original issue. Thanks @itsmeichigo!

**Comment by jaclync** [comment]
I couldn't reproduce this issue with Xcode 13.2.1 in iOS 14 and 15 simulators, I always see the ellipsis menu when navigating to the Reviews tab from the HUB menu 🤔

**Comment by itsmeichigo** [comment]
@jaclync can it be the case that you have loaded the reviews before so the unread notification list has been cached locally - that's why the ellipsis shows up properly? 

My understanding of the case that @jostnes reports here is that the unread list is empty initially, so when the review list is loaded, we ask the navigation item to update but it could not (due to the issue filed in #5830).

**Comment by jaclync** [comment]
> can it be the case that you have loaded the reviews before so the unread notification list has been cached locally - that's why the ellipsis shows up properly?

that's right! I logged out and in again and now I see this issue 👍

**Comment by itsmeichigo** [comment]
Closing this issue since it was fixed in #5882.
